core/src/main/java/org/stellar/anchor/dto/sep38/InfoResponse.java

```diff
@@ -0,0 +1,48 @@
+package org.stellar.anchor.dto.sep38;
+
+import com.google.gson.annotations.SerializedName;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import lombok.Data;
+import org.stellar.anchor.asset.AssetResponse;
+
+@Data
+public class InfoResponse {
+  private List<AssetInfo> assets = new ArrayList<>();
+
+  public InfoResponse(List<AssetResponse> assetResponseList) {
+    assetResponseList.forEach(
+        assetResponse -> {
+          AssetInfo newAssetInfo = new AssetInfo();
+          newAssetInfo.asset = assetResponse.getSchema().toString() + ":" + assetResponse.getCode();
```

Comment:
Can you set attributes directly like this in Java? I thought you needed `newAssetInfo.setAsset(...)`?

Reply:
Done in https://github.com/stellar/java-stellar-anchor-sdk/pull/95/commits/38e84a38fa6293058f7195e53922fbd10605bf26.